packages/test-kit/src/with-feature.ts

```diff
@@ -165,6 +162,12 @@ export function withFeature(withFeatureOptions: IWithFeatureOptions = {}) {
         navigationOptions: suiteNavigationOptions,
     } = withFeatureOptions;
 
+    const {   
+        headless=!process.env.DEBUG,
```

Comment:
if I only pass `devtools: true` from the `withFeature` call, I'd expect `headless` to be implcitly set to `false` (playwright's default behavior). Is us defining a default `true` overrides that behavior?

also, can we use something less generic than `DEBUG`? I'd expect that env variable to possibly affect other executables on some systems.

do we want to allow `DEBUG=false` or `DEBUG=0` to turn that off?

Reply:
I was thinking the default is the current behavior , but no code changed needed for debugging.
changes to the code should override it.
re the env var name: TEST=true ? TEST_DBG=true?
I don't think DEBUG=false or DEBUG=0 should have an effect, only DEBUG=true. I'll adjust the test acordigly